GrievanceManagement/src/main/java/com/grievance/dto/ChangePasswordDto.java

```diff
@@ -0,0 +1,99 @@
+package com.grievance.dto;
+
+/**
+ * Change Password Dto.
+ */
+public class ChangePasswordDto {
+  /**
+   * old password.
+   */
+  private String oldPassword;
+
+  /**
+   * new password.
+   */
+  private String newPassword;
+
+  /**
+   * confirm password.
+   */
+  private String confirmPassword;
+
+  /**
+   * getter for old password.
+   *
+   * @return the oldPassword
+   */
+  public String getOldPassword() {
+    return oldPassword;
+  }
+
+  /**
+   * setter for old password.
+   *
+   * @param old the oldPassword to set
+   */
+  public void setOldPassword(final String old) {
+    this.oldPassword = old;
+  }
+
+  /**
+   * getter for new password.
+   *
+   * @return the newPassword
+   */
+  public String getNewPassword() {
+    return newPassword;
+  }
+
+  /**
+   * setter for new Password.
+   *
+   * @param newPass the newPassword to set
+   */
+  public void setNewPassword(final String newPass) {
+    this.newPassword = newPass;
```

Comment:
Try to follow a convention for getter/setter object names. Eg: oldPassword -> old but newPassword -> newPass